netlify.toml

```diff
@@ -0,0 +1,10 @@
+[build]
+  base = "jekyll"
+  publish = "../docs"
+
+  # Set the baseurl to "/" and url to "$DEPLOY_PRIME_URL" so that the built site is served from the root
+  command = """
+  printf "url: \"$DEPLOY_PRIME_URL\"\nbaseurl: \"/\"" > _config_netlify.yml; \
```

Comment:
Since we want to build new sites for PR preview, the url will vary PR by PR. So this line is needed to dynamically generate suitable Jekyll configs for Netlify.
And because it needs to be dynamic, it can't be a committed `_config_netlify.yml` file.